Source/WebCore/platform/graphics/texmap/BitmapTexture.cpp

```diff
@@ -86,7 +86,11 @@ void BitmapTexture::updateContents(const void* srcData, const IntRect& targetRec
     // We are updating a texture with format RGBA with content from a buffer that has BGRA format. Instead of turning BGRA
     // into RGBA and then uploading it, we upload it as is. This causes the texture format to be RGBA but the content to be BGRA,
     // so we mark the texture to convert the colors when painting the texture.
+#if __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
Use `CPU(LITTLE_ENDIAN)`

Reply:
I changed it.

Would you mind to explain why `CPU(LITTLE_ENDIAN)` is preferred over the various other  ways to handle endianness-conditional code across the WebKit codebase?